_posts/2015-01-30-foreman-community-newsletter-january.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Foreman Community Newsletter - January 2015
+date: '2015-01-30T14:38:00.002+02:00'
+author: Lukáš Zapletal
+tags:
+- newsletter
+modified_time: '2015-01-30T14:38:53.060+02:00'
+blogger_id: tag:blogger.com,1999:blog-6789674575954398874.post-4344260925100932853
+blogger_orig_url: http://blog.theforeman.org/2015/01/foreman-community-newsletter-january.html
+---
+
+<!--more-->
+
+### <span style="background-color: white; color: #333333; font-family: Arial; font-size: 17px; font-style: normal; font-variant: normal; font-weight: bold; text-decoration: none; vertical-align: baseline; white-space: pre-wrap;">Foreman 1.7.2 released</span>
```

Comment:
I've already run it through pandoc once, I think anything further would have to be by hand, annoyingly. It's not on the first page of results, and I doubt the newsletters get high hits after a couple of months. Can we add this to the nitpick list?